internal/adapters/cloudformation/aws/ec2/adatp_test.go

```diff
@@ -0,0 +1,66 @@
+package ec2
```

Comment:
nit: typo in file name, should be `adapt_test.go`

Reply:
fixed https://github.com/aquasecurity/trivy-iac/pull/69/commits/20a2496e11dd6e3e6b958b12c8953a439432b72a